### PR TITLE
Add non-minimal pre-commit config

### PR DESCRIPTION
**Pull request by hmpf**

### CODE REVIEW

**Review by johannaengland — CHANGES_REQUESTED** [review]
Maybe also add black while you're at it?

And the pre-commit should be added to the README.

**Comment by hmpf** [comment]
> Maybe also add black while you're at it?

I wouldn't call a pre-commit with black "minimal" :)

> And the pre-commit should be added to the README.

Willdo.

**Review by johannaengland — CHANGES_REQUESTED** [review]
I was more thinking to do it the way it is mentioned in the Argus README (https://github.com/Uninett/Argus#code-style) for consistency and to also explain very shortly how to enable the pre-commit

**Comment by hmpf** [comment]
I do not have the brains for mind-reading today, and I'm once again out of caffeine. Please update (or close) the PR as you see fit. This is just polish, a nice to have. The important one, #13, has already been merged.

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -5,4 +5,17 @@ repos:
...
+    rev: 3.9.2
+    hooks:
+    -   id: flake8
+        name: "Flake8: critical"
+        args: ['--count', '--select=E9,F63,F7,F82', '--show-source', '--statistics']
+-   repo: https://github.com/psf/black
+    rev: 23.1.0
+    hooks:
+    -   id: black
+        exclude: migrations/
```
Comment (hmpf) [inline .pre-commit-config.yaml]: Ticket-plugins will never have migrations. Then they would be apps and would have to be put into INSTALLED_APPS as well.

**Review by lunkwill42 — APPROVED** [review]

**Pull request merged**